src/components/loader/loader.scss

```diff
@@ -1,6 +1,8 @@
+$faded-black: rgba(0, 0, 0, .8);
```

Comment:
This contradicts the sass rule we have. Running lint prompts that we get rid of leading zeros. It sass-lint expects that we use .8 instead of 0.8. And using 0.8 will prevent commit because of the implementation of pre-commit hooks.